modules/clearCompleted.js

```diff
@@ -4,7 +4,8 @@ import displayToDOList from './displayToDOList.js';
 const clearCompleted = () => {
   const tasks = ToDoList.getTasksList();
 
-  const incompleteTasks = tasks.filter(ToDoList.isInComplete);
+  // const incompleteTasks = tasks.filter(ToDoList.isInComplete);
```

Comment:
Kindly take out the commented code as this would help put you in the habit of writing cleaner and better code